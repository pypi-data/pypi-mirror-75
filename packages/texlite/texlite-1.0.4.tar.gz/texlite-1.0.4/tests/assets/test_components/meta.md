:title: Title Name
:author: Author Name
:date: This is the date
:fontsize: 12pt
:margin: 2.0in
:linespread: 1.3