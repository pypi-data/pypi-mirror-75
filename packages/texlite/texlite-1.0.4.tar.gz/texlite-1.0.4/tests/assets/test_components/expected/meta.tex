% meta
\documentclass[12pt]{extarticle}

% packages
\usepackage[utf8]{inputenc}
\usepackage[margin=2.0in]{geometry}
\usepackage{hyperref}
\usepackage{amsmath}
\usepackage{amssymb}

% preamble commands
\linespread{1.3}
\usepackage{graphicx}
\graphicspath{{tests/assets/test_components/}}

% document details
\title{\textbf{Title Name}}
\author{Author Name}
\date{This is the date}

% ------------------------------------------------------------------------------
\begin{document}
% ------------------------------------------------------------------------------

\maketitle{}

% ------------------------------------------------------------------------------
\end{document}
% ------------------------------------------------------------------------------

